src/inject/dynamic-theme/index.ts

```diff
@@ -318,28 +318,54 @@ function onDOMReady() {
     logWarn(`DOM is ready, but still have styles being loaded.`, loadingStyles);
 }
 
-let documentVisibilityListener: () => void = null;
+/**
+ * The following code contains a workaround for extensions designed to prevent page from knowing when it is hidden
```

Comment:
```suggestion
 *  Begin of Page Lifecycle API workarounds
 *
 * The following code contains a workaround for extensions designed to prevent page from knowing when it is hidden
```